src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -520,7 +520,7 @@ void HeapDumpDCmd::execute(DCmdSource source, TRAPS) {
   // This helps reduces the amount of unreachable objects in the dump
   // and makes it easier to browse.
   HeapDumper dumper(!_all.value() /* request GC if _all is false*/);
-  dumper.dump(_filename.value(), output(), (int) level, _overwrite.value(), (uint)parallel);
+  dumper.dump(_filename.value(), output(), (int)level, _overwrite.value(), (uint)parallel);
```

Comment:
I'm not sure what the prevalent style is for cast operators, but I don't see any point doing this in this PR when the file is otherwise untouched. Thanks.